etc/sysconfig/ceph

```diff
@@ -24,3 +24,11 @@ TCMALLOC_MAX_TOTAL_THREAD_CACHE_BYTES=134217728
 # currently active on the node.
 #
 CEPH_AUTO_RESTART_ON_UPGRADE=no
+
+## unmap all devices, or only those configured in /etc/ceph/rbdmap
+#
+# By default, "rbdmap unmap" (systemctl stop rbdmap) unmounts and unmaps
+# all locally mapped RBD images. Configuring "no" here overrides this
+# behaviour so that only RBD images specified in /etc/ceph/rbdmap are
+# unmounted and unmapped.
+RBDMAP_UNMAP_ALL=yes
```

Comment:
Nit: if this is the default, perhaps leave it commented out as an example of the param